backend/battles/tests/test_forms.py

```diff
@@ -15,24 +15,48 @@ def setUp(self):
 
     def test_create_a_team(self):
         params = {
-            "initial": {"trainer": self.trainer},
             "data": {
-                "pokemon_1": self.pokemon_1,
-                "pokemon_2": self.pokemon_2,
-                "pokemon_3": self.pokemon_3,
+                "trainer": self.trainer.id,
+                "pokemon_1": mommy.make("pokemon.Pokemon", id=1).id,
+                "pokemon_2": mommy.make("pokemon.Pokemon", id=2).id,
+                "pokemon_3": mommy.make("pokemon.Pokemon", id=3).id,
             },
         }
         form = CreateTeamForm(**params)
-        self.assertTrue(form)
+        self.assertTrue(form.is_valid())
 
-    def test_pokemon_exceeds_points_limit(self):
+    def test_team_cant_have_identical_pokemon(self):
         params = {
-            "initial": {"trainer": self.trainer, "battle": self.battle},
             "data": {
+                "trainer": self.trainer.id,
                 "pokemon_1": mommy.make("pokemon.Pokemon", id=1).id,
+                "pokemon_2": mommy.make("pokemon.Pokemon", id=1).id,
+                "pokemon_3": mommy.make("pokemon.Pokemon", id=2).id,
+            },
+        }
+        form = CreateTeamForm(**params)
+        self.assertFalse(form.is_valid())
+
+    def test_team_cant_have_invalid_pokemon(self):
+        params = {
+            "data": {
+                "trainer": self.trainer.id,
+                "pokemon_1": mommy.make("pokemon.Pokemon", id=-5).id,
+                "pokemon_2": mommy.make("pokemon.Pokemon", id=1).id,
+                "pokemon_3": mommy.make("pokemon.Pokemon", id=2).id,
+            },
+        }
+        form = CreateTeamForm(**params)
+        self.assertFalse(form.is_valid())
+
+    def test_pokemon_exceeds_points_limit(self):
```

Comment:
Is this the test that is doing external requests? If yes, this external request needs to be mocked, tests should never make requests to external APIs.